pkg/asset-manager-utils/test/RewardsAssetManager.test.ts

```diff
@@ -147,17 +147,6 @@ describe('Rewards Asset manager', function () {
       ).to.be.revertedWith('Target must be less than or equal to upper critical level');
     });
 
-    it('reverts when setting target above 95%', async () => {
```

Comment:
Let's replace this with a test that sets the target to 100%, and make sure we have a test involving the rebalancing relayer that exits a pool with 0 cash.

Reply:
Would you not say that this is appropriately covered by the tests for checking `upperCriticalPercentage <=100%` and `targetPercentage <= upperCriticalPercentage`?

The rebalancing relayer already tests for a zero cash case we we've set the investment percentage to 100%

Reply:
If we already have such a test that's fine then, my point was we should at least have _a_ test.

Though how could we have a rebalancing relayer test with 100% target before this commit?

Reply:
We didn't, by "already" I mean we bump the percentage from 95% to 100% in this commit.